data/brands/amenity/bank.json

```diff
@@ -46,14 +46,14 @@
       }
     },
     {
-      "displayName": "ABANCA",
+      "displayName": "Abanca",
       "id": "abanca-ce59ab",
       "locationSet": {"include": ["es"]},
       "tags": {
         "amenity": "bank",
-        "brand": "ABANCA",
+        "brand": "Abanca",
         "brand:wikidata": "Q9598744",
-        "name": "ABANCA",
+        "name": "Abanca",
         "official_name": "ABANCA Corporación Bancaria"
```

Comment:
Should this official_name also be changed?

Reply:
According to [its wikipedia](https://en.wikipedia.org/wiki/Abanca), don't. Official name usually comes in uppercase